scripts/check_no_panics.py

```
@@ -251,6 +273,221 @@ def is_test_only_path(path: str) -> bool:
+
+def default_module_candidates(source: pathlib.Path, name: str) -> tuple[pathlib.Path, ...]:
+    if source.name in {"lib.rs", "main.rs", "mod.rs"}:
+        module_dir = source.parent
+    else:
+        module_dir = source.parent / source.stem
+    return module_dir / f"{name}.rs", module_dir / name / "mod.rs"
+
+
+def module_edges(
+    source: pathlib.Path,
+    inherited_test_only: bool,
```

Ruff (0.16.0)

[warning] Boolean-typed positional argument in function definition

(FBT001)